akka-docs/src/main/paradox/stream/operators/Source-or-Flow/monitor.md

```diff
@@ -14,7 +14,35 @@ Materializes to a `FlowMonitor` that monitors messages flowing through or comple
 
```

Comment:
Something looks wrong in the `@apidoc` anchors: `java` is duplicated.

Reply:
Good catch. This is a leftover from the automated update.